src/docs/get-started/editor.md

```diff
@@ -81,7 +84,19 @@ VS Code is a lightweight editor with Flutter app execution and debug support.
  1. Review the output in the **OUTPUT** pane for any issues.
 
 </div>
+<div class="tab-pane" id="vscode" role="tabpanel" aria-labelledby="vscode-tab" markdown="1">
+
+## Install Emacs
+
+Emacs is a lightweight editor with support for Flutter and Dart.
 
+* [Emacs][], latest stable version
+
+## Install the LSP dart package
+
+Check [lsp-dart][] for information on how to install and setup.
```

Comment:
We reverse the sense of sentences like this, per https://developers.google.com/style/clause-order. Maybe:

For information on how to install and use the package, see the 
[lsp-dart documentation][].

It seems to me that the link should be to https://emacs-lsp.github.io/lsp-dart/?